### PR TITLE
feat(accounts): Support retrieving account groups

### PR DESCRIPTION
**Pull request by JTBlanchard**
Add support for retrieving account groups list, specifically to support the
SharedWithAccounts field on tests.

### CODE REVIEW

File: account_groups.go [hunk 1]
```diff
@@ -16,3 +18,32 @@ type SharedWithAccount struct {
...
+	}
+	var target map[string][]AccountGroup
+
+	if dErr := c.decodeJSON(resp, &target); dErr != nil {
+		return nil, fmt.Errorf("Could not decode JSON response: %v", dErr)
+	}
+	if _, ok := target["accountGroups"]; !ok {
+		return nil, fmt.Errorf("'accountGroups' not found in JSON response")
+	}
+	fmt.Printf("%v", target)
```
Comment (william20111) [inline account_groups.go:36]: need to remove the debugging

**Review by william20111 — CHANGES_REQUESTED** [review]

File: account_groups.go (hunk 1 above)
Comment (JTBlanchard) [reply account_groups.go:36]: Done.  Thanks for catching that.

**Comment by william20111** [comment]
This one looks like its failing CI just now

**Comment by JTBlanchard** [comment]
> This one looks like its failing CI just now

Fixed.  My local linting should have caught that.  I'll have to find out why it didn't.

**Pull request merged**